doc/policy/mempool-replacements.md

```diff
@@ -74,3 +76,6 @@ This set of rules is similar but distinct from BIP125.
 
 * RBF enabled by default in the wallet GUI as of **v0.18.1** ([PR
   #11605](https://github.com/bitcoin/bitcoin/pull/11605)).
+
+* Full replace-by-fee enabled as a configurable mempool policy as of **v.0.24.0** ([PR
```

Comment:
```suggestion
* Full replace-by-fee enabled as a configurable mempool policy as of **v24.0** ([PR
```

Reply:
concur

Reply:
Updated.

> concur (wish I could just ACK someone's comment to note I mean the same thing...)

On my side, I generally do a thumbs up to convey I ACK someone's comment.